.travis.yml

```diff
@@ -4,22 +4,19 @@ node_js:
   - 6
   - 8
   - 9
+env:
+  - PYTHON_VERSION=2.7.14
```

Comment:
2.7.14 is the current release here; 2.7.11 is two years old and hasn't been current for a year and a half